src/site/content/en/metrics/fcp/index.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: First Contentful Paint (FCP)
+authors:
+  - philipwalton
+date: 2019-11-07
+updated: 2019-11-07
+description: |
+  This post introduces the First Contentful Paint (FCP) metric and explains
+  how to measure it
+tags:
+  - performance
+  - metrics
+---
+
+{% Aside %}
+  First Contentful Paint (FCP) is an important, user-centric metric for
+  measuring [perceived load speed](/user-centric-metrics/#types-of-metrics)
+  because it marks the first point in the page load timeline that the user can
+  see anything on the screen—a fast FCP helps reassure the user that something
+  is [happening](/user-centric-metric/#user-centric-metric-questions).
+{% endAside %}
+
+## What is FCP?
+
+The First Contentful Paint (FCP) metric measures the time from when the page
+starts loading to when any part of the page's content is rendered on the screen.
+For this metric, "content" refers to text, images (including background images),
+`<svg>` elements, or non-white `<canvas>` elements.
```

Comment:
Is it appropriate to note here that Chrome does not behave properly for non-white canvas?

Reply:
This is from the explainer, so I suppose we should discuss whether or not it should be removed from there.